src/main/java/TicTacToe/Board.java

```diff
@@ -9,10 +11,11 @@ public Board() {
         this.grid = new ArrayList<>(Arrays.asList("1", "2", "3", "4", "5", "6", "7", "8", "9"));
     }
 
-    public void setGrid(List<String> grid) {
-        this.grid = grid;
+    public List<String> getGridAsList() {
```

Comment:
The name of your method describes the data structure that you choose which is a concern only into this class, would it be correct to call it something like `getBoard`?